src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -134,3 +149,21 @@ $table-cell-height-half: $table-cell-height / 2;
   color: $foreground-2;
   cursor: default;
 }
+
+.kd-replicasetdetail-no-events {
+  margin-top: 4 * $baseline-grid;
+  text-align: center;
+}
+
+md-icon {
+  &.kd-replicasetdetail-no-events-icon {
+    color: $muted;
+    font-size: 1.5 * $headline-font-size-base;
```

Comment:
Take font size from _variables.scss or add a new one copied from https://github.com/angular/material/blob/HEAD/src/core/style/variables.scss

$display-1-font-size-base or $display-2-font-size-base should work